### DISCUSSION TITLE
not received coin in my wallet, but it appear in explorer

### DISCUSSION THREAD

**Discussion by siewhong** [Chia-Network/chia-blockchain#4988] [Support]
hi 

my wallet address is 
xch16rvs7njqcxctum3v7u4j39px0et8crj9qvxhv3r95npp4x2elensvkqfnr

I have sent coins to address above, the transaction ID is below, but I didn't receive it in my wallet, even after synced. please help.
https://www.chiaexplorer.com/blockchain/coin/0xaf6ad65730121a9ebbe60669505b37f5ab534e477fd375e91fc5c4caf12a5103

Regards
Sean Lam

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
